notebooks/kubeflow_pipelines/pipelines/solutions/kfp_pipeline_vertex_automl_batch_preds.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>I'd put that explanation just after the pipeline when you explain that you need to create some input data to the pipeline, taking this opportunity to give a brief explanation on that new batch component (probably a word about its input and output and where things are stored is enough).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>